lib/cluster.js

```diff
@@ -8,10 +8,12 @@
  * cluster master.
  */
 
-const cluster = require('cluster');
 const Registry = require('./registry');
 const { Grouper } = require('./util');
 const { aggregators } = require('./metricAggregators');
+// We need to lazy-load the 'cluster' module as some application servers -
+// namely Passenger - crash when it is imported.
+const cluster = () => require('cluster');
```

Comment:
could we do closer to how babel lazy loading works?

e.g.

```js
function cluster() {
  const data = require('cluster');

  cluster = function() {
    return data;
  };

  return data;
}
```

while `require` is cached, avoiding the require-call at all seems prudent

Reply:
Yep I did it that way because `require` is cached, but sure I'll change it